spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/health/SpanCollectorMetricsHandler.java

```diff
@@ -0,0 +1,29 @@
+package org.springframework.cloud.sleuth.health;
+
+public interface SpanCollectorMetricsHandler {
+
+	/**
+	 * Called when spans are submitted to SpanCollector for processing.
+	 *
+	 * @param quantity the number of spans accepted.
+	 */
+	void incrementAcceptedSpans(long quantity);
+
+	/**
+	 * Called when spans become lost for any reason and won't be delivered to the target collector.
+	 *
+	 * @param quantity the number of spans dropped.
+	 */
+	void incrementDroppedSpans(long quantity);
+
+	/**
+	 * Returns the number of accepted spans
+	 */
+	long getAcceptedSpans();
```

Comment:
I don't think we should be exposing getters. real metrics systems will export these metrics, either to a local endpoint (along with other metrics of use) or outside the process. The concern of collecting metrics doesn't need to be conflated with returning them